packages/node_modules/@webex/plugin-meetings/src/common/errors/parameter.js

```diff
@@ -12,10 +12,15 @@ export default class ParameterError extends Error {
   */
   constructor(message = ERROR_DICTIONARY.PARAMETER.MESSAGE, error = null) {
     super(message);
+
+    if (Error.captureStackTrace) {
```

Comment:
Can this change be in its own commit? since it doesn't relate to local video quality changes?

Reply:
Yes, this was actually from another branch that got in here somehow during a rebase.